moler/device/abstract_device.py

```diff
@@ -167,3 +191,22 @@ def start(self, cmd_name, **kwargs):
         :param kwargs: dict with parameters for command constructor.
         :return: command object
         """
+
+    def register_handler_to_notify_to_forget_device(self, handler):
+        """
+        Registers handler to notify when device should be forgot.
```

Comment:
Registers callable to be called (notifed) when device is removed

Reply:
Done.